example/src/app/ng-reflect-as-text/ng-reflect-as-text.component.html

```diff
@@ -0,0 +1,3 @@
+<p>
+  ng-reflect-as-text works!
```

Comment:
Do you mind adding `ng-reflect` in an attribute value here? Like `<p some-attr="ng-reflect-as-attr-arg works">`

Reply:
Done